examples/python_sdk/concurrent-example.py

```diff
@@ -0,0 +1,53 @@
+import os
+import backoff
+
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from snowflake.snowpark import Session
+from snowflake.core.exceptions import APIError
+from snowflake.core import Root
+from dotenv import load_dotenv
+
+# Setup exponentioal backoff on "too many requests" errors (status: 429), with
+# up to 10 retries.
+@backoff.on_exception(backoff.expo,
+                      APIError,
+                      max_time=60, # seconds
+                      max_tries=10,
+                      giveup=lambda x : x.status != 429)
+def runQuery(service, query):
+    return service.search(query=query, columns=["LYRIC","ALBUM_NAME","TRACK_TITLE","TRACK_N","LINE"], limit=1)
```

Comment:
Could we parameterize the column names, since these will be dependent on the service?
Or, we can just omit columns altogether (`columns=[]`)

Reply:
done.